packages/wdio-cli/src/launcher.js

```diff
@@ -342,16 +348,16 @@ class Launcher {
         }
         this.interface.emit('job:end', { cid, passed, retries })
 
-        // Update schedule now this process has ended
-        if (!this.isMultiremote) {
-            // get cid (capability id) from rid (runner id)
-            cid = parseInt(cid, 10)
+        /**
```

Comment:
Seems like the code below will now get executed for both multi remote as well as regular mode, is this correct?

Reply:
absolutely! There is no more difference here now between regular and multiremote modes